### PR TITLE
Add link to AEP Swift SDKs

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,11 @@ __Important__ - _all_ AEP SDK Android binaries depend on `core-1.x.x.aar`.
 
 ## Installing the AEP SDK for iOS
 
+> __Important__ 
+> 
+> ⚠️ A newer version of the [AEP Mobile SDKs (3.x)](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift) is now available! These SDKs are open source, bring many exciting new features and include support for Swift Package Manager (SPM), CocoaPods and XCFrameworks.
```
Comment (kevinlind) [inline README.md:73]: As this message is already under the "Important" heading, I don't think the emoji is needed, unless it is added to all the "Important" messages.

File: README.md [hunk 2]
```diff
@@ -68,6 +68,11 @@ __Important__ - _all_ AEP SDK Android binaries depend on `core-1.x.x.aar`.
...
+> If you are currently using our Objective-C (ACP-prefix) libraries, please see the [Migrating to Swift](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift) guide for next steps.
+
 > __Important__
```
Comment (kevinlind) [inline README.md:76]: Can these two "Important" messages get merged into one?
Also, can the first sentence of the second important message be removed or clarified?  It refers only to the AEP SDK, but the ACP SDK is also distributed using XCFrameworks.

**Review by kevinlind — COMMENTED** [review]
Looks good. Added a few small suggestions.

File: README.md (hunk 1 above)
Comment (emdobrin) [reply README.md]: Made few updates so that this note stands out, let me know what you think.

File: README.md (hunk 2 above)
Comment (emdobrin) [reply README.md]: This entire page refers to AEP SDKs, did some renaming to indicate the instructions are mainly for 2.x

File: README.md [hunk 3]
```diff
@@ -66,12 +66,14 @@ In order to do a manual installation of the AEP SDK `.aar` binaries, complete th
...
+## Installing the AEP SDK (2.x) for iOS
 
-> __Important__ 
+> ⚠️ __Important__ 
 > 
-> Newer versions of the AEP SDK for iOS are distributed using XCFrameworks. If using the XCFramework distribution, Xcode 12 or newer is required and Cocoapods 1.10.0 or newer is required (where applicable).
+> A newer version of the [AEP Mobile SDKs (3.x)](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift) is now available! These SDKs are open source, bring many exciting new features and include support for Swift Package Manager (SPM), CocoaPods and XCFrameworks.
+> If you are currently using our Objective-C (ACP-prefix 2.x) libraries, please see the [Migrating to Swift](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift) guide for next steps.
 
+__Important__ - Newer versions of the AEP SDK for iOS are distributed using XCFrameworks. If using the XCFramework distribution, Xcode 12 or newer is required and Cocoapods 1.10.0 or newer is required (where applicable).
```
Comment (kevinlind) [inline README.md:76]: nit: Use "latest" instead of "newer" here, "The latest versions of the AEP SDK ..."

**Review by kevinlind — APPROVED** [review]
Your latest changes look good. Added a minor 'nit' comment.

**Review by praschetan — APPROVED** [review]

**Pull request merged**